package.json

```diff
@@ -39,7 +39,19 @@
     "vitest": "^3.2.4"
   },
   "engines": {
-    "node": "^20.0.0",
-    "npm": "^10.0.0"
+    "node": "^24 || ^22 || ^20",
+    "npm": "^10 || ^11"
```

Comment:
```suggestion
```

Reply:
No need for the NPM dependency it is not required at all as we do not ship the lock file.

Reply:
Why?